addon/mixins/axis-titles.js

```diff
@@ -11,10 +11,31 @@ import Ember from 'ember';
  */
 const AxisTitlesMixin = Ember.Mixin.create({
   /**
-   * Toggle axis titles on/off
+   * Toggle axis X title on/off
    * @type {Boolean}
    */
-  hasAxisTitles: false,
+  hasXAxisTitle: false,
+
+  /**
+   * Toggle axis Y title on/off
+   * @type {Boolean}
+   */
+  hasYAxisTitle: false,
+
+  /**
+   * A deprecated property to support backward compatability.
+   * @deprecated
+   */
+  hasAxisTitles: Ember.computed('hasXAxisTitle', 'hasYAxisTitle',
```

Comment:
Good idea.

We can use `Ember.deprecate`, but we would need to strip out those statements in the build process.